openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -25,8 +25,11 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
         "comment_template": "{comment}",
     }
 
+    # private
```

Comment:
not needed with `_` prefix

Reply:
Removed in https://github.com/Tilix4/OpenPype/pull/78/commits/791c25c98fc65300094d0d3d420157492f984c2a